crmsh/corosync.py

```diff
@@ -460,9 +487,11 @@ def config(self):
         p.set('quorum.device.net.tie_breaker', self.tie_breaker)
         if self.cmds:
             p.add('quorum.device', make_section('quorum.device.heuristics', []))
-            p.set('quorum.device.heuristics.mode', 'sync')
+            mode = self.mode if self.mode else 'sync'
```

Comment:
I guess we could put `sync` as default in `add_argument`, right? (commented below again)